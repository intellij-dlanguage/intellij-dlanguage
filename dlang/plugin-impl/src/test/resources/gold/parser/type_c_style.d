//no longer valid according to dmd:

//int (*fp)(int);  // fp is pointer to a function
