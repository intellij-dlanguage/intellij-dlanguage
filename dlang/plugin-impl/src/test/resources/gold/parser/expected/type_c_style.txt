D Language File
  PsiComment(DlangTokenType.LINE_COMMENT)('//no longer valid according to dmd:')
  PsiComment(DlangTokenType.LINE_COMMENT)('//int (*fp)(int);  // fp is pointer to a function')